### DISCUSSION TITLE
ToggleMark

### DISCUSSION THREAD

**Discussion by godwin5** [sibiraj-s/ngx-editor#336] [Q&A] [answered]
How to use commands.toggleMark(parameter)??

What are the possible parameter to be passed in toggleMark()??

**Comment by sibiraj-s** [comment, marked as answer]
I will update the docs. The command can take one of the keys of marks object, here https://github.com/sibiraj-s/ngx-editor/blob/master/src/schema/marks.ts#L144 as a parameter.

[a comment by an automated account is not shown]
